src/components/connection-prompt/bluetooth/ConnectBatteryDialog.svelte

```diff
@@ -21,6 +22,14 @@
   <div class="space-y-5">
     <p>{$t('connectMB.connectBattery.subtitle')}</p>
     <img src={microbitConnectedImage} alt="" class="w-68 h-229px m-auto" />
+    <a
+      class="inline-flex mr-auto gap-x-1 items-center text-link outline-none focus-visible:ring-4 focus-visible:ring-offset-1 focus-visible:ring-ring"
+      href="https://support.microbit.org/support/home"
```

Comment:
TODO: update link.

Reply:
TODO added in code [b29a676](https://github.com/microbit-foundation/ml-trainer/pull/245/commits/b29a676557add0e4f30a3394e1f59f38b5bf9a99)